src/mapping/stability_measures_accumulator.jl

```diff
@@ -172,15 +160,9 @@ function reset_mapper!(a::StabilityMeasuresAccumulator)
     ds = referenced_dynamical_system(a.mapper)
     Dims = dimension(ds)
     Datatype = eltype(current_state(ds))
-    a.basin_points = Dict{Int, StateSpaceSet{Dims,Datatype}}()
-    a.finite_time_basin_points = Dict{Int, StateSpaceSet{Dims,Datatype}}()
-    a.nonzero_measure_basin_points = Dict{Int, StateSpaceSet{Dims,Datatype}}()
-    a.mean_convergence_time = Dict{Int, Float64}()
-    a.maximal_convergence_time = Dict{Int, Float64}()
-    a.mean_convergence_pace = Dict{Int, Float64}()
-    a.maximal_convergence_pace = Dict{Int, Float64}()
-    a.convergence_times = Dict{Int, Vector{Float64}}()
-    a.convergence_paces = Dict{Int, Vector{Float64}}()
+    a.u0s = Vector{StateSpaceSet{Dims,Datatype}}()
```

Comment:
Why is this a vector of state space sets? It should just be a vector of points?

Reply:
Just a vector? what `u0` is already.  you can transform to a statespaceset at the end, no need to store it